lib/interface/missing_directories_screen.dart

```diff
@@ -407,63 +407,70 @@ class _MissingDirectoriesScreenState extends State<MissingDirectoriesScreen>
               ),
               resizeToAvoidBottomInset: true,
               body: NowPlayingBarScrollHideNotifier(
-                child: CustomScrollView(
-                  controller: controller,
-                  slivers: [
-                    SliverAppBar(
-                      foregroundColor:
-                          Theme.of(context).appBarTheme.foregroundColor,
-                      backgroundColor:
-                          Theme.of(context).appBarTheme.backgroundColor,
-                      leading: IconButton(
-                        onPressed: () {},
-                        icon: Icon(Icons.arrow_back),
-                      ),
-                      expandedHeight: 200.0,
-                      snap: false,
-                      pinned: true,
-                      floating: false,
-                      forceElevated: true,
-                      title: FadeTransition(
-                        opacity: visible,
-                        child: Text(Language.instance.FOLDERS_NOT_FOUND),
-                      ),
-                      flexibleSpace: FlexibleSpaceBar(
-                        background: Padding(
-                          padding: const EdgeInsets.symmetric(horizontal: 16.0),
-                          child: Column(
-                            mainAxisAlignment: MainAxisAlignment.end,
-                            crossAxisAlignment: CrossAxisAlignment.start,
-                            children: [
-                              Text(
-                                Language.instance.FOLDERS_NOT_FOUND,
-                                style: Theme.of(context)
-                                    .textTheme
-                                    .titleLarge
-                                    ?.copyWith(
-                                      fontSize: 24.0,
-                                    ),
-                              ),
-                              const SizedBox(height: 12.0),
-                              Text(
-                                Language.instance.FOLDERS_NOT_FOUND_SUBTITLE
-                                    .replaceAll('\n', ' '),
-                                style: Theme.of(context).textTheme.displaySmall,
+                child: CustomListView(
+                  children: [
+                    CustomScrollView(
```

Comment:
`CustomScrollView` inside `CustomListView`. Why?

Reply:
CustomListView() is the widget having a padding for when miniplayer is sticky and is used pretty much all over the app, except for some pages like these